lib/order.rb

```diff
@@ -0,0 +1,48 @@
+class Order
+
+  def initialize (menu_klass = Menu.new, order_summary_klass = OrderSummary.new, sms_klass = SMS.new)
```

Comment:
Metrics/LineLength: Line is too long. [101/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)